xpti/README.md

```diff
@@ -0,0 +1,3 @@
+# XPTI proxy library
+
+Implementation of the instrumentation stub library to support SYCL Instrumentation. the XPTI framework library has the implementation and will be used by tool developers to extract the traces from SYCL applications. One the proxy library is necessary for building the SYCL runtime library.
```

Comment:
Since the document is in Markdown format, I usually don't pay attention to line lengths as it auto-formats the text. I will try to update the document limiting it to 80 column lines :)

Reply:
Checking in the documents after addressing these issues with long lines and typos.

Reply:
Done!